taccsite_cms/static/site_cms/css/src/_imports/_tests/postcss-preset-env.css

```diff
@@ -0,0 +1,53 @@
+/*
+Test
+
+Styles for testing plugins, e.g.:
+
+- `postcss-preset-env`
+
+Styleguide _Test
+*/
+
+/*! Custom Media Queries */
+/* SEE: https://preset-env.cssdb.org/features#custom-media-queries */
+
+/*! Goal: */
+@media (max-width:30em){._test{background-color:red}}
+
+/*! Test: */
+@custom-media --narrow-window (max-width: 30em);
+
+@media (--narrow-window) {
+  ._test {
+    background-color: red;
+  }
+}
```

Comment:
step 4: `Ensure output matches what the files reports the output should be.`

I am a little confused as to what I am looking for here.

After running step 3 in my local env (outside the containers), I got this in the local `_test.css` file:

```
/*! Custom Media Queries */

/*! Goal: */@media (max-width:30em){._test{background-color:red}}

/*! Test: */@media (max-width:30em){._test{background-color:red}}

/*! Media Query Ranges */

/*! Goal: */@media (max-width:479px){._test{background-color:red}}@media (min-width:480px) and (max-width:767px){._test{background-color:#00f}}@media (min-width:768px){._test{background-color:#0f0}}

/*! Test: */@media (max-width:479px){._test{background-color:red}}@media (min-width:480px) and (max-width:767px){._test{background-color:#00f}}@media (min-width:768px){._test{background-color:#0f0}}

/*! Custom Media Queries & Media Query Ranges */

/*! Goal: */@media (min-width:992px) and (max-width:1199px){._test{background-color:red}}

/*! Test: */@media (min-width:992px) and (max-width:1199px){._test{background-color:red}}
```

Running the steps inside the container gave me no errors but also did not result in a `_test.css` file.

In all cases, the `npm run build` step produced these warnings:

```
37:9    ⚠  variable --global-color-warning--normal is undefined and used without a fallback [postcss-css-variables]
38:9    ⚠  variable --global-color-warning--weak is undefined and used without a fallback [postcss-css-variables]
```

I think I may need moe detailed testing instructions to understand what the expected behavior is?

Reply:
**✓ Tests Passed:**
- The Goal is `@media (max-width:30em){._test{background-color:red}}`.
- The Test output was `@media (max-width:30em){._test{background-color:red}}`.

The warnings are known issue for a temporary feature: [FP-531: Quick: Explain expected warning](https://github.com/TACC/Core-CMS/commit/2524ee9fa36dba3e741fec9831d20f8cb97bc7b8).

Reply:
To help, I have added instructions to the file: [GH-171: Explain how to use the test file (`d221c83`)](https://github.com/TACC/Core-CMS/pull/173/commits/d221c83b4f830b7a3c21053f5c97b99f64ecc9ea).

Suggestions welcome. Clearer instruction? Better file name? Remove tests entirely?¹

> ¹ The tests are only for devs as a sanity check. I experienced problems with PostCSS plugin configuration, when I was new, because (a) certain combinations of _other_ plugins require they be applied in a certain order, otherwise the output is wrong and (b) certain plugins require configuration that is not well-documented for those unfamiliar with the plugin mechanics.

Reply:
Bueno! Made sense after that addition =).